docker_dev.sh

```diff
@@ -3,7 +3,7 @@ docker build -f alpine.docker -t diginc/pi-hole:alpine .
 docker tag diginc/pi-hole:alpine diginc/pi-hole:latest
 docker build -f debian.docker -t diginc/pi-hole:debian .
 
-IP=$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)
+IP=$(ip route get 8.8.8.8 | awk '{ print $NF; exit }')
```

Comment:
I see. This change simply finds what interface traffic to the outside world will egress from. Any network setup that changes this, like a vpn tunnel that changes the default route, will throw it. 

However, I think overall it makes it easier and more consistent for the general use cases of these scripts.